### PR TITLE
Improve Restaurant model

### PR DESCRIPTION
**Pull request by kroyagis**

### CODE REVIEW

File: models/restaurants.go [hunk 1]
```diff
@@ -3,19 +3,22 @@ package models
...
-	ID      int            `json:"id"`
-	Name    string         `json:"name"`
-	Address string         `json:"address"`
-	Number  sql.NullString `json:"number"`
+	ID        int            `json:"id"`
+	Name      string         `json:"name"`
+	Address   string         `json:"address"`
+	Number    sql.NullString `json:"number"`
+	CreatedAt time.Time      `json:"created_at"`
+	UpdatedAt time.Time      `json:"updated_at"`
```
Comment (kroyagis) [inline models/restaurants.go:16]: It seems like the order of these attributes matter

Comment (lilosir) [reply models/restaurants.go:16]: Why do we need the time format? Type string is working,  and I don't know what is going to be when you send it to the front end. I am guessing your database is not correct or different than mine.

Comment (lilosir) [reply models/restaurants.go:16]: Screen Shot 2020-03-26 at 9 38 09 PM
Screen Shot 2020-03-26 at 9 38 16 PM

Comment (kroyagis) [reply models/restaurants.go:16]: Time is time. `time` format gives us control over how we want to format it. 

https://golang.org/pkg/time/

**Comment by kroyagis** [comment]
@lilosir As per our conversation, I'm merging this in. Let me know if you have any issues.

**Pull request merged**